benchmark/galsim/galsim_field.py

```diff
@@ -11,20 +11,22 @@
 FITS_CATALOG_FILENAME = os.path.join('output', 'galsim_field_500_catalog.fits')
 ARCSEC_PER_DEGREE = 3600.
 FIELD_EXPAND_ARCSEC = 10
+SKY_LEVEL_NMGY = 0.155 # similar to SDSS
+COUNTS_PER_NMGY = 180.0 # similar to SDSS
 
 def set_image_dimensions(test_case, catalog_rows):
     min_ra_deg = min(float(row['right_ascension_deg']) for row in catalog_rows)
     max_ra_deg = max(float(row['right_ascension_deg']) for row in catalog_rows)
     min_dec_deg = min(float(row['declination_deg']) for row in catalog_rows)
     max_dec_deg = max(float(row['declination_deg']) for row in catalog_rows)
 
-    width_arcsec = (max_ra_deg - min_ra_deg) * ARCSEC_PER_DEGREE + 2 * FIELD_EXPAND_ARCSEC
-    height_arcsec = (max_dec_deg - min_dec_deg) * ARCSEC_PER_DEGREE + 2 * FIELD_EXPAND_ARCSEC
+    height_arcsec = (max_ra_deg - min_ra_deg) * ARCSEC_PER_DEGREE + 2 * FIELD_EXPAND_ARCSEC
+    width_arcsec = (max_dec_deg - min_dec_deg) * ARCSEC_PER_DEGREE + 2 * FIELD_EXPAND_ARCSEC
```

Comment:
In hindsight, maybe "column" and "row" would have been better than "height" and "width". Unless the columns of the CCD become the rows in Julia image data structure. If that's the case, then any terms would be better than "column" and "row"...